src/Storage/Repository/IInstanceRepository.cs

```diff
@@ -23,11 +23,10 @@ public interface IInstanceRepository
         /// <summary>
         /// Get an instance for a given instance id
         /// </summary>
-        /// <param name="instanceOwnerPartyId">the instance owner id</param>
         /// <param name="instanceGuid">the instance guid</param>
         /// <param name="includeElements">whether to include data elements</param>
         /// <returns>The instance for the given parameters</returns>
-        Task<(Instance Instance, long InternalId)> GetOne(int instanceOwnerPartyId, Guid instanceGuid, bool includeElements = true);
+        Task<(Instance Instance, long InternalId)> GetOne(Guid instanceGuid, bool includeElements = true);
```

Comment:
Avoid default values in cases where the number of call points are manageable.

Reply:
The must interesting thing to address related to this is how to maintain the two instance properties lastchanged and readstatus. In many use cases the data elements are fetched only to make sure that lastchanged and readstatus are set the same way as with the Cosmos repo. The lastchanged semantics is not obvious - the value in the json doc in the db is not the value used externally. One solution could be to always update the two properties directly on the instance for every data element write operation. Then the semantics of the includeDataelements parameter on GetOne would be obvious.